src/ng/directive/form.js

```diff
@@ -204,7 +224,7 @@ function FormController(element, attrs, $scope, $animate, $interpolate) {
         delete object[property];
       }
     },
-    parentForm: parentForm,
+    parentForm: form.$$parentForm,
```

Comment:
This property is not used, since you are accessing the parent form via `ctrl.$$parentForm` inside `addSetValidity()`.